products/eap/get-started.adoc

```diff
@@ -49,13 +49,11 @@ c.  Click _Add Repository_ and select the
 *redhat-techpreview-all-repository* profile and then click Ok. +
 +
 [.content-img]
-image://static.jboss.org/rhd/pr/165/build/372/images/images_products_devstudio_devstudio-overview-1-1.png[Add
-Maven Repository]
+image://static.jboss.org/rhd/pr/165/build/372/images/images_products_devstudio_devstudio-overview-1-1.png[Add Maven Repository]
 d.  Click Finish +
 +
 [.content-img]
-image://static.jboss.org/rhd/pr/165/build/372/images/images_products_devstudio_devstudio-overview-2-1.png[Configure
-Maven Repository]
+image://static.jboss.org/rhd/pr/165/build/372/images/images_products_devstudio_devstudio-overview-2-1.png[Configure Maven Repository]
```

Comment:
Notice that http://static.jboss.org/rhd/pr/165/build/372/images/images_products_devstudio_devstudio-overview-2-1.png no longer exists